internal/auth0/user/data_source_test.go

```diff
@@ -12,58 +12,64 @@ import (
 
 const testAccDataSourceUser = `
 resource "auth0_role" "owner" {
-	name        = "owner"
-	description = "Owner"
+	name        = "Test Owner {{.testName}}"
```

Comment:
All created roles need to have the "Test" keyword within them so they are correctly picked up by the sweeper.